packages/transformers/babel/src/babel7.js

```diff
@@ -66,6 +79,27 @@ export default async function babel7(
     },
   };
 
+  if (applicationProfiler.enabled) {
+    config.wrapPluginVisitorMethod = (
+      key: string,
+      nodeType: string,
+      fn: Function,
+    ) => {
+      return function () {
+        let measurement;
+        if (applicationProfiler.enabled) {
```

Comment:
Can the profiler be disabled mid-build or is the inner check unnecessary?

Reply:
Ah yep, I can remove this inner check.